usr/lib/common/object.c

```diff
@@ -444,6 +444,47 @@ CK_BBOOL object_is_session_object(OBJECT * obj)
         return TRUE;
 }
 
+// object_is_extractable()
+//
+CK_BBOOL object_is_extractable(OBJECT * obj)
+{
+    CK_BBOOL val;
+    CK_RV rc;
+
+    rc = template_attribute_get_bool(obj->template, CKA_EXTRACTABLE, &val);
+    if (rc != CKR_OK)
+        return TRUE;
```

Comment:
Should it really return TRUE if the CKA_EXTRACTABLE attribute is not there? Since the attributes default is TRUE that might be OK though.

Reply:
Yes, that was my intention